yabt/builders/golang.py

```diff
@@ -122,6 +126,25 @@ def go_test_builder(build_context, target):
     go_builder_internal(build_context, target, command='test')
```

Comment:
I think you need to change the command here to 'build'. I think that when both build func and test func are registered ybt will first run the build and then run the test.

Reply:
I don't think it works, since go expects in 'build' a main function, but not in test. and the '-c' flag to only compile the test if specific to 'test', not to build. I'll try though.

Reply:
Tried. 'build' with -c crashes since this flag is only supported in 'go test'. Without '-c' it crashes since there is no main to build. As far as I could find out, 'go test -c' is the way to build a test binary without executing the tests.